quirklike/Assets/AI/Standard Scripts/EnemyStats.cs

```diff
@@ -6,22 +6,28 @@
 //the future but do i care? yes 
 //but it is a problem for future me to worry about
 
-public class EnemyStats : MonoBehaviour {
+public class EnemyStats : MonoBehaviour, IDamageable 
```

Comment:
What is the difference between EStats and EnemyStats? like what purpose do they each have

Reply:
Enemystats is not suppsoed to be there. Estats is the new version. I tried changing Enemystats but decided to write a new script instead